pexpect/async.py

```diff
@@ -30,10 +31,15 @@ def __init__(self, expecter):
     def found(self, result):
         if not self.fut.done():
             self.fut.set_result(result)
+            self.transport.pause_reading()
```

Comment:
Ha! Well, I guess you were pre-emptively ensuring that the machinery to solve our problem was in place ;-)